File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/ShippiingAddressFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.payments.ShippingAddress;

public class ShippiingAddressFilterValidator extends FilterValidator<ShippingAddress> {

    public ShippiingAddressFilterValidator(CustomValidator<ShippingAddress> validator) {
        super(ShippingAddress.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/ShippiingAddressFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.payments.ShippingAddress
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class ShippiingAddressFilterValidator(validator: Validator<ShippingAddress>) : FilterValidator<ShippingAddress>(
    ShippingAddress::class, validator)
